trace/Sampling.md

```diff
@@ -0,0 +1,32 @@
+# Sampling
+
+This document is about the sampling bit, sampling decision, samplers and how and when does 
+OpenCensus sample traces. A sampled traces is a traces that gets exported via the configured
+exporters.
+
+## Sampling Bit (propagated via TraceOptions)
+
+Sampling bit is always set only at the start of a Span, and is using a `Sampler`
+
+### What kind of samplers does OpenCensus support?
+* `AlwaysSample` - sampler that always makes a "yes" decision every time.
+* `NeverSample` - sampler that always makes a "no" decision every time.
+* `Probability` - sampler that tries to uniformly sample traces with a given probability. When 
+applied to a child `Span` of a **sampled** parent `Span` then it keeps the sampling decision.
+
+### How can users control the Sampler that is used for sampling?
+There are 2 ways to control the `Sampler` used when the library does sampling:
+* Controlling the global default `Sampler` via [TraceConfig](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/TraceConfig.md).
+* Pass a specific `Sampler` when start the [Span](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/Span.md)
+
+### When does OpenCensus sample traces?
+The OpenCensus library does sampling based on the following rules:
+1. If the span is a root `Span` then a `Sampler` will be used to make the sampling decision:
+   * If user provided a `Sampler`, use it to determine the sampling decision.
+   * Else use the global default `Sampler` to determine the sampling decision.
+2. If the span is a child of a remote `Span` the sampling decision will be:
```

Comment:
we should mention how someone can retain the remote decision as this is common practice in internal network tracing (and all B3 for example)

Reply:
Not sure what you mean here? I think I mentioned that ProbabilitySampler retain the sampling decision (AlwaysSampler same). NeverSample does not retain is a kind of force "unsampled".